pine64/pinebook-pro/firmware/ap6256-firmware/default.nix

```diff
@@ -10,6 +10,7 @@ let
     rev = "421b7dd8f3c67f66910710838a0be03f3575a3c9";
     sha256 = "175qcjfaz7nhpyh0hxiih53k3hly407lkpxgissvldghxrw01ccn";
   };
+  clm_blob = ./brcmfmac43456-sdio.clm_blob;
```

Comment:
Can you just add the file in question to `rkwifibt` as well? Than we can keep this repo blob free.